### ISSUE TITLE
HOTA higher than the result in your paper

### ISSUE THREAD

**Issue by github2016-yuan** [dyhBUPT/StrongSORT#64]
Nice repo and I follow your cmds to eval strongsort++ on MOT17, but I get much higher result than data in your paper.
Here are details:
First I download from [baidu](https://pan.baidu.com/s/1EtBbo-12xhjsqW5x-dYX8A?pwd=sort#list/path=%2F)
SO I get **MOT17-val_results** file folder and **MOT17-train** file folder. 

Then I use [TrackEval](https://github.com/JonathonLuiten/TrackEval)  to evaluate the results on the MOT17-val set.
`python scripts/run_mot_challenge.py \
--BENCHMARK MOT17 \
--SPLIT_TO_EVAL train \
--TRACKERS_TO_EVAL xxx/MOT17-val_results/StrongSORT++ \
--TRACKER_SUB_FOLDER '' \
--METRICS HOTA CLEAR Identity VACE \
--USE_PARALLEL False \
--NUM_PARALLEL_CORES 1 \
--GT_LOC_FORMAT '{gt_folder}/{seq}/gt/gt_val_half_v2.txt' \
--OUTPUT_SUMMARY False \
--OUTPUT_EMPTY_CLASSES False \
--OUTPUT_DETAILED False \
--PLOT_CURVES False`

Then I get result below:

```
HOTA: data-pedestrian              HOTA      DetA      AssA      DetRe     DetPr     AssRe     AssPr     LocA      OWTA      HOTA(0)   
MOT17-02-FRCNN                     52.775    51.035    55.512    58.37     72.925    61.375    74.464    83.922    56.704    64.31        
MOT17-04-FRCNN                     80.459    78.038    83.272    84.406    85.395    87.277    89.107    88.702    83.804    91.916    
MOT17-05-FRCNN                     62.77     65.422    60.431    71.684    78.977    75.762    69.857    84.524    65.809    76.756       
MOT17-09-FRCNN                     70.954    72.508    69.509    75.074    87.434    76.539    82.051    87.213    72.231    82.91        
MOT17-10-FRCNN                     62.696    59.913    66.132    64.582    76.413    70.937    79.37     81.036    65.304    81.782        
MOT17-11-FRCNN                     68.75     63.992    73.967    74.896    77.274    78.102    88.555    88.982    74.427    78.623       
MOT17-13-FRCNN                     68.072    65.264    71.278    70.074    79.983    76.891    81.684    83.687    70.65     84.825       
COMBINED                           70.778    67.658    74.573    74.526    81.179    79.945    84.673    86.64     74.481    83.318    
```

Here is result in your paper:
StongSORT++ : HOTA is 64.4 and it is lower than the resut I get(**70.778 - 64.4 = 6.378**)
Something wrong with my work? Thanks.

**Comment by dyhBUPT** [comment]
Hi, thanks for your interest in our repo.

Your results are for MOT17 validation set, while "64.4" is for MOT17 test set.

**Comment by github2016-yuan** [comment]
Ok thanks.
